### ISSUE TITLE
Expose info regio website

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio.eu#307]
https://linkedopendata.eu/wiki/Property:P1742 for all projects concerned

**Comment by madewild** [comment]
Thank you @DiaZork, @svili will take it from here. 
However, the program url is currently
https://ec.europa.eu/regional_policy/opempl/detail.cfm?cci=2014FR05M0OP001&lan=en

and I thought it should be
https://ec.europa.eu/regional_policy/en/atlas/programmes/2014-2020/france/2014fr05m0op001

@roberto-musmeci @bisiaam could you confirm which one we should use?

**Comment by Bisiaam** [comment]
Yes you have all the OPS here: https://ec.europa.eu/regional_policy/en/atlas/programmes

**Comment by madewild** [comment]
@DiaZork could you change the urls to the atlas? don't know where the old one comes from...

**Comment by roberto-musmeci** [comment]
Maybe just a few words to clarify.

All programs financed by REGIO Funds (e.g. ERDF, CF) are availalbe in InfoRegio
https://ec.europa.eu/regional_policy/en/atlas/programmes

Programs finanaced by the EMPL funds (e.g. ESF) alone are intested in DG EMPL portal only. 
This is why we have two possible urls - _I am  trying to find the relevant post in github._ 

Programs financed both by REGIO and EMPL funds - such as in the example made by Max in https://github.com/ec-doris/kohesio.eu/issues/307#issuecomment-1021423627 (funded by ERDF and ESF) - the summary of the program is in both REGIO and EMPL websites. 

Bottom line, when possible we use the DG REGIO page because the style is better. Content is basically the same.

**Comment by madewild** [comment]
OK very clear, thank you @roberto-musmeci !

**Comment by madewild** [comment]
The inforegio programme website is there but not the inforegio **project** website.
@svili can you check the api? we can discuss the correct way to add this...

**Comment by D063520** [comment]
It's in the api. For example https://dev.kohesio.eu/projects/Q139379
<img width="1680" alt="Screenshot 2022-01-27 at 14 20 21" src="https://user-images.githubusercontent.com/10883585/151367028-51a0ff06-7b6d-4b3c-8b19-e47bb0d55677.png">

**Comment by madewild** [comment]
Yes yes indeed, only a front-end problem (the issue was not very clear and mixing two things, sorry about that)

**Comment by madewild** [comment]
This is done, if we need to tweak you can mention it in #277
